piet/src/text.rs

```diff
@@ -114,6 +114,11 @@ pub trait Text: Clone {
 ///
 /// This has a `'static` bound because the inner type will be behind a shared
 /// pointer.
+///
+/// # Implementors
+///
+/// Implementors of this trait must not mutate the text after creation. `TextLayout`s using this
+/// `TextStorage` will be set up for the old text and so will be in an inconsistent state.
```

Comment:
```suggestion
/// This trait expects immutable data. Mutating the data (using interior mutability)
/// May cause any [`TextLayout`] objects using this [`TextStorage`] to become
/// inconsistent.
```

... sorry I'm a bit hyper atm ➿

Reply:
Don't apologise for your great work!